app/models/recipe.rb

```diff
@@ -3,4 +3,5 @@ class Recipe < ApplicationRecord
   validates :price, numericality: { only_integer: true }, allow_nil: true
   validates :cooking_time, numericality: { only_integer: true, greater_than:0 }, allow_nil:true
   validates :video_link, format: URI::DEFAULT_PARSER.make_regexp(%w[http https]), allow_nil: true
+  validates :level, numericality: {only_integer: true, greater_than_or_equal_to:1, less_than_or_equal_to:5}, allow_nil: true
```

Comment:
```suggestion
  validates :level, numericality: { only_integer: true, in: 1..5 }, allow_nil: true
```
https://guides.rubyonrails.org/active_record_validations.html#numericality